tests/test_integration_pytest_extension.py

```diff
@@ -1,3 +1,9 @@
+import pytest
+
+pytest_version = tuple(int(v) for v in pytest.__version__.split("."))
+
+
+@pytest.mark.skipif(pytest_version < (6, 0), reason="at least pytest 6 required")
```

Comment:
What part of this test is pytest 6 dependent? Is there a way to widen the support?

Reply:
`from_parent` doesn't exist on pytest.Item

Reply:
I actually recall changing this when upgrading to pytest 6. The older method fails in pytest 6, part of the suite of deprecations